### ISSUE TITLE
Status Message "NEW_NOTIFICATION" instead of actual notification

### ISSUE THREAD

**Issue by devz3r0** [nextcloud/android#7469]
### Steps to reproduce
1. install NC app
2. connect to one (or in my case more nextcloud) server(s)
3.  wait

### Expected behaviour
nothing, or the actual notitification

### Actual behaviour
Status bar message NEW_NOTIFICATION 

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: 10

Device model: oneplus 7t pro

Stock or customized system: stock

Nextcloud app version:
3.14.0 RC2 

Nextcloud server version:
the servers are 18.x and 19.x
Reverse proxy:
one nginx, the other one apache, the third one none I think. 

#### Web server error log
```
nothing found as I don't know from what nextcloud server is that, because the app isn't telling
```

#### Nextcloud log (data/nextcloud.log)
```
If the app tells me the server, then I can probably paste the logs
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by strobeltobias** [comment]
I have the same problem.
Duplicate of #1964 ?

**Comment by mr2017x** [comment]
same issue arrives on 24.0.1 with actual nc client on android.

**Comment by BlackWolfWoof** [comment]
> same issue arrives on 24.0.1 with actual nc client on android.

Same on 24.0.6
![Screenshot_20221018-103206](https://user-images.githubusercontent.com/29355595/196379164-7f74f8d3-dbc8-4ae1-a711-d461b19ac402.jpg)

**Comment by CaesarRulezzz** [comment]
I've this issue with the latest build from the Play Market

**Comment by origintopleft** [comment]
Same issue here, latest from Play Store. Connected to a single NextCloud 24.0.7 instance (I'm waiting on app updates before moving to 25), running via docker-compose with nginx as a reverse proxy on Fedora 36. Will post log data when I get home, I'm currently on my way to an appointment. I should note that according to Android, NEW_NOTIFICATION pretty consistently comes through the "Miscellaneous" category.

*(Note to self: 14:45)*

**Comment by hugalafutro** [comment]
Excuse me, but with this going on for 2 years how is it not a high priority issue? It is still happening on android 13 and latest stable nextcloud. 

As I understand it Nextcloud is touted as a business cloud solution as well, are you telling me the businesses who use it and presumably pay for it are perfectly fine with their employees receiving confusing notifications with zero informational content?

![Screenshot_2022-12-22-12-00-42-40_8142442b4340f41fb3bd44535ad8f76b](https://user-images.githubusercontent.com/30209689/209130479-bbcc6691-a9a3-4326-854f-ea54ba0c73a6.jpg)

**Comment by AlvaroBrey** [comment]
> Moreover how is it closed?

Because it's a duplicate of https://github.com/nextcloud/android/issues/1964, which is open.

**Comment by origintopleft** [comment]
Good news - I just got another one (I forgot to post logs earlier, I apologize) and remembered I had the desktop client running this time! The notification source is an app on NextCloud having an update available.

**Comment by origintopleft** [comment]
> > Moreover how is it closed?
> 
> Because it's a duplicate of #1964, which is open.

It is *not* a duplicate of #1964. I actually went and read it. #1964 was caused by issues sending data to the push proxy. This is caused by something else. Just because it says "NEW_NOTIFICATION" again does *not* mean it's the same bug. That's like assuming every time your leg hurts is because you broke it the first time when you were 12 years old, when in this case it's because you have old beat up shoes. Yes, it *feels* the same, but it is not the same problem.

**Comment by BlackWolfWoof** [comment]
> Excuse me, but with this going on for 2 years how is it not a high priority issue? Moreover how is it closed? It is still happening on android 13 and latest stable nextcloud.
> 
> As I understand it Nextcloud is touted as a business cloud solution as well, are you telling me the businesses who use it and presumably pay for it are perfectly fine with their employees receiving confusing notifications with zero informational content?
> 
> ![Screenshot_2022-12-22-12-00-42-40_8142442b4340f41fb3bd44535ad8f76b](https://user-images.githubusercontent.com/30209689/209130479-bbcc6691-a9a3-4326-854f-ea54ba0c73a6.jpg)

This solved for me when i used the Nextcloud apk from the playstore instead of fdroid. 
I am guessing because of the way they both push notiifcations but i am definetly not an expert in this so take this with a grain of salt. 
*Also sorry for the late reply*

**Comment by Popkornium18** [comment]
Deleting the app data and setting up the app again solved this for me.

**Comment by hugalafutro** [comment]
@Popkornium18 Thanks for the suggestion, I am now getting the correct notification, but for w/e reason in triplicate:
![image](https://user-images.githubusercontent.com/30209689/227158231-e217ff31-1388-4876-936d-ecce36811c1e.png)
